src/components/RefinementList/__tests__/__snapshots__/RefinementList-test.js.snap

```diff
@@ -47,6 +48,7 @@ exports[`RefinementList rendering with facets 1`] = `
         dangerouslySetInnerHTML={
           Object {
             "__html": Object {
+              "attribute": "attribute",
```

Comment:
is this correct?

Reply:
Yes, we now pass the `attribute` too to the template to fill the `name` HTML property:

https://github.com/algolia/instantsearch.js/blob/feat/3.0-numeric-menu/src/components/RefinementList/RefinementList.js#L48

https://github.com/algolia/instantsearch.js/blob/feat/3.0-numeric-menu/src/widgets/numeric-menu/defaultTemplates.js#L4